logging/logging.go

```diff
@@ -250,69 +276,78 @@ func (instance *Logging) getLog(request getLogRequest) {
 		}
 
 		if err = archInstance.addLog(createLogString(logEntry, needUnitField)); err != nil {
-			if err == errMaxPartCount {
+			if errors.Is(err, errMaxPartCount) {
 				log.Warn(err)
 				break
 			}
 
-			err = aoserrors.Wrap(err)
-
-			return
+			return err
 		}
 	}
 
-	if err = archInstance.sendLog(request.logID); err != nil {
-		err = aoserrors.Wrap(err)
-		return
-	}
+	return nil
 }
 
-func (instance *Logging) getServiceCrashLog(request getLogRequest) {
-	var err error
-
-	// error handling
-	defer func() {
-		if err != nil {
-			log.Error("Can't get service crash logs: ", err)
-
-			instance.sendErrorResponse(err.Error(), request.logID)
+func (instance *Logging) getInstanceCrashLog(request getLogRequest) (err error) {
+	journal := SDJournal
+	if journal == nil {
+		if journal, err = sdjournal.NewJournal(); err != nil {
+			return aoserrors.Wrap(err)
 		}
-	}()
-
-	var journal *sdjournal.Journal
-
-	journal, err = sdjournal.NewJournal()
-	if err != nil {
-		err = aoserrors.Wrap(err)
-		return
 	}
 	defer journal.Close()
 
-	if err = instance.addUnitFilter(journal, request.serviceID); err != nil {
-		err = aoserrors.Wrap(err)
-		return
+	if err = instance.addUnitFilter(journal, request.instanceIDs); err != nil {
+		return err
 	}
 
 	if request.till == nil {
 		if err = journal.SeekTail(); err != nil {
-			err = aoserrors.Wrap(err)
-			return
+			return aoserrors.Wrap(err)
 		}
 	} else {
 		if err = journal.SeekRealtimeUsec(uint64(request.till.UnixNano() / 1000)); err != nil {
-			err = aoserrors.Wrap(err)
-			return
+			return aoserrors.Wrap(err)
 		}
 	}
 
 	var crashTime uint64
 
+	crashTime, err = instance.getCrashTime(journal, request.from)
+	if err != nil {
+		return err
+	}
+
+	if crashTime == 0 {
+		return
```

Comment:
Empty return. I suggest use anonymous return value in this and other functions.

Reply:
accep